src/net/connection/virtual_connection.rs

```diff
@@ -109,6 +109,17 @@ impl VirtualConnection {
 
         Ok(Some(Packet::new(self.remote_address, Box::from(payload), header.delivery_method)))
     }
+
+    /// This will gather dropped packets from the reliable channels.
+    ///
+    /// Note that after requesting dropped packets the dropped packets will be removed from this client.
+    pub fn gather_dropped_packets(&mut self) -> Option<Vec<Box<[u8]>>>{
+        if self.reliable_unordered_channel.has_dropped_packets() {
+            return Some(self.reliable_unordered_channel.drain_dropped_packets());
```

Comment:
no returns please 😄

Reply:
Clippy lint should be catching these, and if we aren't running it is as part of the CI/CD, we should

Reply:
That's a personal style thing, lots of people prefer returns. However, no returns is as close to an axiom as you'll find in Rust style I think, so I'm fine with keeping them out. Let's just automate it.

Reply:
I am actually prefering return is most cases. But in this scenario I ended up with a compiler error. I'll revisit it and see what I can do.

Reply:
I believe you can't in this case because of the if structure.

Reply:
@TimonPost if you want we can look at it together in a play.rust-lang.org

Reply:
There you are: https://play.rust-lang.org/?version=stable&mode=debug&edition=2015&gist=faaec323b3e789cd878cf1092fe2b4a8

Reply:
Calling this one resolved. Make an issue for it if someone feels it needs to be revisited.